src/test/scala/nl/knaw/dans/easy/bagindex/components/DatabaseSpec.scala

```diff
@@ -79,11 +78,12 @@ class DatabaseSpec extends BagIndexDatabaseFixture with Database {
       DateTime.parse("2017-03-09"),
       DateTime.parse("2018")
     )
+    val dois2 = List("10.5072/dans-a1b-cd2e", "10.5072/dans-f3g-hi45", "10.5072/dans-j6k-lm78", "10.5072/dans-n9o-pq01", "10.5072/dans-r2s-tu34")
 
     List(
-      (bagIds1.zip(times1), baseId1),
-      (bagIds2.zip(times2), baseId2))
-      .flatMap { case (xs, base) => xs.map { case (bagId, time) => addBagInfo(bagId, base, time) }}
+      ((bagIds1, times1, dois1).zipped.toList, baseId1),
+      ((bagIds2, times2, dois2).zipped.toList, baseId2))
+      .flatMap { case (xs, base) => xs.map { case (bagId, time, doi) => addBagInfo(bagId, base, time, doi) }}
```

Comment:
I am a bit lost here, but you do want to call addBagInfo for a lot of 'test vectors' I guess.